Resources/Prototypes/Recipes/Lathes/tools.yml

```diff
@@ -112,5 +112,23 @@
   result: RCDAmmo
   completetime: 600
   materials:
-    steel: 500
-    plastic: 250
+    Steel: 500
+    Plastic: 250
+
+- type: latheRecipe
+  id: Mop
+  icon:
+    sprite: Objects/Specific/Janitorial/mop.rsi
+    state: mop
+  result: MopItem
+  completetime: 500
+  materials:
+    Steel: 200
+    Plastic: 100
+
```

Comment:
remove these extra empty lines at the end of the file

Reply:
ok